Modules/_elementtree.c

```diff
@@ -899,6 +905,8 @@ deepcopy(elementtreestate *st, PyObject *object, PyObject *memo)
 
     if (Py_REFCNT(object) == 1) {
         if (PyDict_CheckExact(object)) {
+            // Exact dictionaries do not execute arbitrary code as it's
```

Comment:
@serhiy-storchaka is this assumption correct? namely, here I don't need to incref `object` temporarily right?

Reply:
`PyDict_Next()` does not use `__iter__`, so this comment is redundant. `PyDict_Next()` does not call any user code.